edge/pkg/edged/edged_status.go

```diff
@@ -24,6 +24,7 @@ package edged
 
 import (
 	"fmt"
+	"github.com/kubeedge/beehive/pkg/core"
```

Comment:
Please sorts the import here.

Reply:
Ok I sort the import.

Reply:
Thanks, would you like to squash the commits?

Reply:
Done